.github/workflows/manifold.yml

```diff
@@ -21,25 +21,56 @@ jobs:
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-    - uses: actions/checkout@v2
-    - uses: jwlawson/actions-setup-cmake@v1.12
-    
     - name: Install dependencies
       run: |
         apt-get -y update
-        DEBIAN_FRONTEND=noninteractive apt install -y libglm-dev libassimp-dev libcgal-dev
-    - name: Build CUDA
+        DEBIAN_FRONTEND=noninteractive apt install -y libcgal-dev libomp-dev nodejs git python3
```

Comment:
As I'm looking at this more, I'm thinking maybe we'd be better off with some of these builds as separate actions, rather than separate jobs. The WASM build is the most obvious example to me; no need for CUDA, but requires nodejs, etc. I might even split out the tools build that uses CGAL, since that's unrelated to this library. 

This way, when someone is looking to build, they have concise instructions for only their situation. Also, we'll get feedback on their failures separately in Actions. Anyway, this doesn't necessarily have to happen right now, just something to think about.